samples/matter/smoke_co_alarm/README.rst

```diff
@@ -0,0 +1,379 @@
+         * Blinking in combination with LED1 and LED3 when other alarms are triggered. 
+
+      LED 3:
+         The following states are possible:
+
+         * Blinking evenly 300 ms on/300 ms off when the battery low level alarm is expressed.
+         * Blinking in combination with LED1 and LED2 when other alarms are triggered. 
+
+      LED 1, LED 2 and LED 3 combined:
+         The following states are possible:
+
+         * All three LEDs linking at the same time oddly 300 ms on/700 ms off when the hardware fault alarm is expressed.
+         * All three LEDs linking at the same time oddly 700 ms on/300 ms off when the end of service alarm is expressed.
+         * Flushing in sequence from LED1, through LED2 to LED3 with 200 ms interval when the self-test is triggered.
+
+      Button 0:
+         .. include:: /includes/matter_sample_button.txt
+
+      Button 2:
+         Functions as the User Active Mode Trigger (UAT) button.
+         For more information about Intermittently Connected Devices (ICD) and User Active Mode Trigger, see the :ref:`ug_matter_device_low_power_icd` documentation section.
+
+      .. include:: /includes/matter_segger_usb.txt
+
+NFC port with antenna attached:
+    Optionally used for obtaining the `Onboarding information`_ from the Matter accessory device to start the :ref:`commissioning procedure <matter_light_bulb_sample_remote_control_commissioning>`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/matter/smoke_co_alarm`
+
+.. include:: /includes/build_and_run.txt
+
+See `Configuration`_ for information about building the sample with the DFU support.
+
+Selecting a custom configuration
+================================
+
+Before you start testing the application, you can select one of the :ref:`matter_smoke_co_alarm_custom_configs`.
+See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information how to select a configuration.
+
+Testing
+=======
+
+After building the sample and programming it to your development kit, complete the following steps to test its basic features.
+
+.. note::
+   The following steps use the CHIP Tool controller as an example.
+   Visit the :ref:`test event triggers <ug_matter_test_event_triggers>` page to see the code values used for triggering specific alarms. 
+
+.. tabs::
+
+   .. group-tab:: nRF52 and nRF53 DKs
+
+      #. |connect_kit|
+      #. |connect_terminal_ANSI|
+      #. If the device was not erased during the programming, press and hold **Button 1** until the factory reset takes place.
+      #. Commission device to the Matter network.
+         See `Commissioning the device`_ for more information.
+      #. Observe that **LED 2**, **LED 3** and **LED 4** are turned off, which means that the device does not express any alarm.
+      #. Trigger the device self-test by invoking the following command with the <node_id> and <endpoint_id> replaced with values selected by you (for example ``1`` and ``1``):
```

Comment:
```suggestion
      #. Trigger the device self-test by invoking the following command with the *<node_id>* and *<endpoint_id>* replaced with values selected by you (for example, ``1`` and ``1``):
```
We could also do what is done with the [testing remote access](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/matter/lock/README.html#testing_remote_access_with_pin_code_credential) instructions for the door lock sample, and give a filled out command and explain the two values after the command.

Also, I think the values need to be consistent from step to step, right? If so, that needs to be clarified.

Reply:
TBH I didn't want to do that as in the door lock sample. I have impression that if you give a filled out command, the user usually is not going to read the explanation, but will just copy-paste the example values. 

The values of the node_id and test_event_enable_key have to be the same for all commands related to `generaldiagnostics`, but the first one applies to different endpoint_id. Because of that it was not filled, but given as variable (for `generaldiagnostics` it has to be 0, for others like smokecoalarm it can be whatever suits to the user)

Reply:
Alright, that makes sense. I'd add the information information on what needs to be the same to the start of each tab, before the steps, and have a suggestion to select them there with the example values. That way, you don't have to keep repeating the examples values in each step, and can instead say "replaced with the values you selected".